src/screens/Settings/index.tsx

```diff
@@ -5,19 +5,23 @@ import CodeMirror from '@uiw/react-codemirror'
 import { useCallback, useEffect, useRef } from 'react'
 import styled from 'styled-components'
 
-import { CODE_MIRROR_OPTIONS, CODE_MIRROR_THEME } from './constants'
 import { Button } from '../../elements/Button'
 import { useCachedState } from '../../hooks/useCachedState'
 import { Screen } from '../../layouts/Screen'
-import { Webview, type Core } from '../../types'
+import { type Core, Webview } from '../../types'
+import { CODE_MIRROR_OPTIONS, CODE_MIRROR_THEME } from './constants'
 
 export function Settings() {
   const clamdConfFileSourceRef = useRef<string | null>(null)
 
   const [state, setState, updateState] = useCachedState<Core.SettingsState>(Webview.CacheKey.SETTINGS_STATE, {
+    // biome-ignore lint/style/useNamingConvention: Core event data.
     clamd_conf_file_path: null,
+    // biome-ignore lint/style/useNamingConvention: Core event data.
     clamd_conf_file_source: null,
+    // biome-ignore lint/style/useNamingConvention: Core event data.
     is_ready: false,
+    // biome-ignore lint/style/useNamingConvention: Core event data.
```

Comment:
I don't have a choice here since these types represent core event body (data) which uses snake case as convention (among other reasons, the core is in Rust and Rust uses snake case).

The only other option is to either ignore the camel case check for the entire project which I definitely don't want or to ignore this entire file (including all the other rules) in the project Bione config which I definitely don't want either.